README.md

```diff
@@ -200,6 +200,8 @@ If you would like to use an OpenWhisk action to conveniently produce a message t
 |topic|String|The topic you would like the trigger to listen to|
 |value|String|The value for the message you would like to produce|
 |key|String (Optional)|The key for the message you would like to produce|
+|base64DecodeValue|Boolean (Optional - default=false)|If true, the message will be produced with a base64 decoded version of the value parameter|
```

Comment:
Should probably capitalize all the occurrences of `base64` in this file to match the existing doc style.